flask web frontend/app.py

```
@@ -0,0 +1,876 @@
+import logging
+import time
+import re
+import json
+import traceback
+import sys
+from uuid import uuid4
+from functools import partial
+
+import threading
+from logging.handlers import RotatingFileHandler
+import time
```

Ruff (0.14.6)

Redefinition of unused `time` from line 9

Remove definition: `time`

(F811)